oam/install_scripts/mcs-primproc.service.in

```diff
@@ -1,14 +1,14 @@
 [Unit]
 Description=mcs-primproc
 PartOf=mcs-workernode.service
-After=mcs-workernode.service
+After=mcs-controllernode.service
```

Comment:
Included this to prevent a race condition where exemgr ends up finishing before controllernode. My previous change introduced the break in the chain that caused primproc (and everything after primproc) to race against controllernode

Reply:
PartOf=mcs-workernode is still the correct dependency because on the non-primary nodes mcs-controllernode will not exist but it'd be nice if we could still have a chain on restarts/stops from workernode all the way down to ddlproc.